tests/testutil.h

```diff
@@ -201,12 +201,12 @@ static void NN_UNUSED test_drop_impl (char *file, int line, int sock, int err)
     }
 }
 
-static int get_test_port(int argc, const char *argv[])
+static int NN_UNUSED get_test_port (int argc, const char *argv[])
```

Comment:
Ok, I see now.  I think this is goofy, and but its doing what was intended, because testutil.h is compiled inline and these are static functions.  So I'm fine with that for now.  We ought to fix this to be less "weird" in the future.  (A separate header and .c file would be more typical.)